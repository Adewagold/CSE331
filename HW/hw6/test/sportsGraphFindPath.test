# Tests find minimum-cost path on sports graph.

# Load sports.tsv
LoadGraph sports sports.tsv

# Test find minimum-cost path not shortest path
FindPath sports Michael Howard