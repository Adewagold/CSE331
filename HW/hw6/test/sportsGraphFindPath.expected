# Tests find minimum-cost path on sports graph.

# Load sports.tsv
loaded graph sports

# Test find minimum-cost path not shortest path
path from Michael to Howard:
Michael to Jeff with weight 0.333
Jeff to Howard with weight 0.333
total cost: 0.667